package.json

```diff
@@ -78,7 +78,9 @@
     "nanoid": "3.1.31",
     "simple-get": "4.0.1",
     "minimatch": "5.1.0",
-    "semver": "7.5.3"
+    "semver": "7.5.3",
+    "get-func-name": "2.0.1",
```

Comment:
Since these weren't listed before, they must have been transitive dependencies. Which package(s) were they being inherited from, and can we update those dependencies instead? 

Otherwise, future developers may wonder why these transitive dependencies are listed here. So at a minimum, the commit message should explain why they have been added - e.g. "get-func-name is normally inherited by package abc, but to address CVE 123, we are now explicitly stating a dependency on it so we can use the patched version".